lib/gds_api/publishing_api_v2.rb

```diff
@@ -106,6 +106,24 @@ def publish(content_id, update_type = nil, options = {})
     post_json(publish_url(content_id), params)
   end
 
+  # Republish a content item
+  #
+  # The publishing-api will "republish" a republishable_item item, so that it will set a live edition
```

Comment:
you've got "republishable_item item" here.

I'd suggest rephrasing to: The publishing-api will "republish" a live edition. This can be used to remove an unpublishing or to re-send a published edition downstream"

Reply:
sure